### DISCUSSION TITLE
[SingleFile Companion) Syntax of the save path on Windows

### DISCUSSION THREAD

**Discussion by itby28** [gildas-lormeau/SingleFile#959] [General]
Hello. installed a companion (windows, firefox), in the file options.json indicated the path to save to an existing directory on the disk C "savePath": C:\Tmp\", but nothing is saved to this directory, it is saved to some other place that I don't know.. What is the correct way to specify the path in this settings file? Now the settings file looks like this:

```
{
    "browserHeadless": true,
    "browserDebug": false,
    "savePath": C:\Tmp\
}
```

**Comment by gildas-lormeau** [comment]
The content of your settings file is not valid JSON. Here is below the valid content.
```json
{
  "browserHeadless": true,
  "browserDebug": false,
  "savePath": "C:\\Tmp\\" 
}
```
